src/main/resources/application.yml

```
@@ -1,8 +1,17 @@
 spring:
+  h2:
+    console:
+      enabled: true
+      path: '/h2-console'
   liquibase:
     change-log: classpath:/db/changelog.yml
   application:
     name: cwa-quick-test-backend
+  jpa:
+    generate-ddl: true
+    hibernate:
+      ddl-auto: update
```

This is not suitable for the master branch :D
```suggestion
    hibernate:
      ddl-auto: validate
```